js/shipment.js

```diff
@@ -86,6 +86,14 @@ export const GO_GREEN_LOGO =
 
 export const ShipmentAPI = superclass =>
     class extends superclass {
+        /**
+         * Creates a shipment and associated DHL waybill.
+         *
+         * @param {Object} payload The payload object according to the DHL API standards.
+         * @param {Object} options An object of options to configure the request.
+         * @returns {Object} The HTTP response object.
+         * @see https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section/operations-pickup-exp-api-pickups

```

Comment:
Notice that you copy-pasted the pickup one and this is the shipment method. So at the very least would be a different URL. Regardless, let's use "https://developer.dhl.com/api-reference/dhl-express-mydhl-api#reference-docs-section" for the same reason as before.